apps/ledger-live-mobile/src/screens/MyLedgerDevice/AppsList/Searchbar.tsx

```diff
@@ -40,6 +42,8 @@ export default ({ searchQuery, onQueryUpdate }: Props) => {
         returnKeyType="search"
         maxLength={50}
         onChangeText={onQueryUpdate}
+        onFocus={onFocus}
```

Comment:
I'm not sure if the component has treated the `undefined` case, if not, maybe checking the type before passing is better.

Reply:
It's working 👍